RailwayCo/Assets/Scripts/GameLogic/Currency/CurrencyManager.cs

```diff
@@ -5,7 +5,8 @@
 
 public class CurrencyManager : ICloneable, IEquatable<CurrencyManager>
 {
-    public Dictionary<CurrencyType, int> CurrencyDict { get; private set; }
+    [JsonProperty]
+    private Dictionary<CurrencyType, int> CurrencyDict { get; set; }
```

Comment:
`private set` to be explicit?

Naming convention too

Reply:
A `private` property will have `private` getter and setter, there is no need to specify `private set`, or rather you can't, because there will be [compilation error](https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs0273?f1url=%3FappId%3Droslyn%26k%3Dk(CS0273)) as it is of equivalent level of restriction and not more than the property's.